variables.tf

```diff
@@ -8,8 +13,89 @@ variable "location" {
   type        = string
 }
 
+variable "log_analytics_workspace_id" {
+  description = "The ID of the Log Analytics workspace to send diagnostics to."
+  type        = string
+}
+
+variable "administrator_login" {
+  description = "The login username of the administrator for this MySQL server."
+  type        = string
+}
+
+variable "databases" {
+  description = "A list of databases to create on the MySQL Flexible Server."
+  type = map(object({
+    name      = string
+    collation = optional(string, "utf8_general_ci")
+    charset   = optional(string, "utf8")
+  }))
+}
+
+variable "server_version" {
+  description = "The version of this MySQL server"
+  default     = "5.7"
```

Comment:
Version 5.7 is marked as "retired" in the Azure Portal:

![image](https://github.com/user-attachments/assets/bae7eec7-3469-453d-8a5c-1a98f96ee602)

According to Azure provider docs, the only other possible value is `8.0.21`:

```suggestion
  default     = "8.0.21"
```